pkg/oc/lib/describe/describer.go

```diff
@@ -685,14 +687,21 @@ func DescribeImage(image *imagev1.Image, imageName string) (string, error) {
 			formatAnnotations(out, image.ObjectMeta, "")
 		}
 
+		if err := ocimageutil.ImageWithMetadata(image); err != nil {
+			return err
+		}
+		dockerImage, ok := image.DockerImageMetadata.Object.(*dockerv10.DockerImage)
+		if !ok {
+			return fmt.Errorf("unable to read image metadata")
```

Comment:
nit: misleading... we were able to read image metadata, but we were not able to convert them to appropriate object? probably showing what is the source will help debug this :)

Reply:
Will fix.